src/log/log.c

```diff
@@ -95,11 +95,14 @@ __wt_log_background(WT_SESSION_IMPL *session, WT_LSN *lsn)
 int
 __wt_log_force_sync(WT_SESSION_IMPL *session, WT_LSN *min_lsn)
 {
+	struct timespec fsync_start, fsync_stop;
 	WT_DECL_RET;
 	WT_FH *log_fh;
 	WT_LOG *log;
+	uint64_t fsync_duration_usecs;
 
 	log = S2C(session)->log;
+	fsync_duration_usecs = 0;
```

Comment:
We generally don't initialize variables unless we need to. Also variable initialization blocks are usually alphabeticalized (i.e: the order would be swapped here, so `log` followed `fsync..`